js/webtest_js.mli

```diff
@@ -7,4 +7,6 @@ val setup : Webtest.Suite.t -> unit
 (** [setup test] sets up a test runner and attaches it to the document's onLoad
     handler. *)
 
+val run : ?with_colors:bool -> Webtest.Suite.t -> unit
```

Comment:
Added comment in the new commit. The idea is to compile test with `let () = Webtest_js.Runner.run suite` to byte code, convert it to js and run it with node.